src/ConnectedMode/Suppressions/SuppressedIssueMatcher.cs

```diff
@@ -70,6 +73,11 @@ private static bool IsMatch(IFilterableIssue issue, SonarQubeIssue serverIssue)
                 return false;
             }
 
+            if (!IsSameFile(issue, serverIssue))
```

Comment:
Strictly speaking the method is also checking whether the issues are both module-level (i.e. don't have a file). I couldn't think of a better name, though.